### ISSUE TITLE
expose ui when blackduck is not enabled

### ISSUE THREAD

**Issue by yashbhutwala** [blackducksoftware/synopsys-operator#771]
**What happened**:
When blackduck is not enabled, and expose ui is run, the log statements indicate the a ui is available now,  but it's not.  A route is made, but a pod for ui is not added to the deployment. This might not matter once we move to a python based ui, but logging the issue here.

**What you expected to happen**:
Just create an error saying "UI is not available for Alert deployment right now, cannot use expose-ui when only alert crd is enabled"

**How to reproduce it (as minimally and precisely as possible)**:
```sh
./synopsysctl-6-2 deploy --enable-alert -n so-alert
./synopsysctl-6-2 update operator -n so-alert --expose-ui OPENSHIFT

kc get po -n so-alert
synopsys-operator-84cdb84677-7pkz4   1/1     Running   0          25m

kc get routes -n so-alert
synopsys-operator-ui   synopsys-operator-ui-so-alert.10.1.176.94.xip.io          synopsys-operator   synopsys-operator-ui   edge          None
```

**Comment by msenmurugan** [comment]
@yashbhutwala can you please verify it again?

**Comment by yashbhutwala** [comment]
@msenmurugan still able to replicate it, using `release-2019.8.x` branch

**Comment by yashbhutwala** [comment]
Fixed by #989
